src/countryEnum.tsx

```diff
@@ -0,0 +1,246 @@
+export enum Country {

```

Comment:
```suggestion
// ISO 3166-1 alpha-2
export enum Country {

```

I think `Add country enum support` is really great idea! 👍🏻
just additionally, how about mentioning https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2 as well?